src/app/store.js

```diff
@@ -13,7 +13,7 @@ function rehydrateInitialState() {
       options: null,
       grid: [],
       frames: [],
-      endEvent: {},
+      endEvent: null,
```

Comment:
Yeah, I'm unsure about this. I think it's okay to make this change (it's certainly rational)... however...

We might want to toggle the scrubber on GAME_OVER, instead of RECEIVE_EVENT_END. 
* GAME_OVER is fired when the last frame is actually drawn
* RECEIVE_EVENT_END is fired when the last frame is loaded into the buffer (but not yet shown).

So maybe we actually want a new boolean in the state to indicate the GAME_OVER event has fired at least once. Maybe `gameHasEnded`?

You've tested this more than I have, what do you think UX wise?

Reply:
> what do you think UX wise?

I think `RECEIVE_EVENT_END` is actually preferred for a UX, since `GAME_OVER` means the User had to watch the whole game before they could scrobble right? [Didn't actually implement so let me know if that does't seem correct!]
Having to wait for the game to run once first seems to decrease the usefullness of the bar for quickly reviewing games, and especially for jumping to the end of long games

Happy to try it out if we want to see if that way, but I think leaving it on `RECEIVE_EVENT_END` has better UX [though it does mean the bar 'pops in' a few seconds after the game has loaded initially]

Reply:
Sounds good, I'm game to try it. My thinking is it'll pop in at unpredictable times, depending on game age and length, which could look/feel odd. 

As a separate future thought - we should put an indicator somewhere as to whether or not the game has fully buffered, like a red/yellow/green icon. That might help give context to the scrubber popping in.

Reply:
> As a separate future thought - we should put an indicator somewhere as to whether or not the game has fully buffered, like a red/yellow/green icon. That might help give context to the scrubber popping in.

Ya that might be good!

Another idea is maybe the space given to the scrubber should always be there, so even when it does pop in it doesn't reflow other elements 🤔 
In that spirit maybe it goes right under the media controls, and could always take up space 🤷 
Happy to play with that if you like it or leave it for another slice!

Reply:
> I do wonder if, scrubber is enabled, show it but keep it gray (or fixed at 0) until max turn is available?

Ohh ya I like this! Will try that and see what it looks like